PathingAPI/PPather/Triangles/MPQTriangleSupplier.cs

```diff
@@ -213,9 +217,9 @@ public string SetZone(string zone)
             for (int i = 0; i < maps.recordCount; i++)
             {
                 int mapID = maps.GetInt(i, 0);
-                // logger.WriteLine("   ID:" + maps.GetInt(i, 0));
-                // logger.WriteLine(" File: " + maps.GetString(i, 1));
-                // logger.WriteLine(" Name: " + maps.GetString(i, 4)); // the file!!!
+                logger.WriteLine("   ID:" + maps.GetInt(i, 0));
+                logger.WriteLine(" File: " + maps.GetString(i, 1));
+                logger.WriteLine(" Name: " + maps.GetString(i, 4)); // the file!!!
```

Comment:
undo this change. it would rather just spam the output

Reply:
the main reason why I did this to have a chance to see the location in order to fix it if it not found.